src/prefab.ts

```diff
@@ -91,11 +99,19 @@ export const prefab = {
   },
 
   isEnabled(key: string): boolean {
-    return this.get(key) === true;
+    const value = this.get(key) === true;
+
+    this.trackEvaluation(key, value, this.context);
```

Comment:
We shouldn't need this `trackEvaluation` call since this function uses `get` and `get` has its own.

Reply:
Oh, ha, oops. Well that will cut the chattiness in half 😅